rts/Rendering/Env/Particles/ProjectileDrawer.cpp

```diff
@@ -619,10 +629,12 @@ void CProjectileDrawer::DrawProjectileNow(CProjectile* pro, bool drawReflection,
 		return;
 
 	// no-op if no model
-	DrawProjectileModel(pro);
+	if (DrawProjectileModel(pro))
```

Comment:
Could this be a standalone change? It looks unrelated to all the MT stuff so could get in independently.

Reply:
I didn't test it extensively, but it looked like an inherited spring bug.
Maybe it can go standalone indeed.